src/help.go

```diff
@@ -142,6 +142,7 @@ const (
 	DescOpen               = "open a file in the appropriate filemanager or default browser"
 	DescUrl                = "returns the url of each file"
 	DescVerbose            = "show step by step information verbosely"
+	DescFixClashes         = "fix clashes by renaming files. WARNING - this may be racy"
```

Comment:
In later iterations, we'll add an etag in the request so that we can lock on that version of a file if multiple clients are operating on that file. If the Etag and or name don't match then that file is surely gone. That's the mechanism we'll use for curbing data racing. It is also mentioned as a TODO in the README